src/commands/android/index.ts

```diff
@@ -194,6 +195,11 @@ export class AndroidSetup {
 
       Logger.log(prelude + ' ' + colors.grey(desc));
     });
+
+    if (this.options.help) {
+      Logger.log();
+      showSubcommandHelp();
```

Comment:
Should be moved to the `run()` method itself.

The functions should ideally be atomic and do just one thing. Calling another function from within a function makes it more difficult to follow the code.

Also, the `showHelp` method name could be modified to `showOptionsHelp` to better represent what this method does.